### PR TITLE
Added class to checkbox and wrapper div around volunteer preference a…

### PR DESCRIPTION
**Pull request by mfung**
…dditional info, added jquery show/hide

This should close issue 232 if I understand it correctly.

### CODE REVIEW

**Comment by tjgrathwell** [comment]
Hi @mfung, this is good but it doesn't account for the 'edit' case.

When an RSVP form is rendered for edit, your `.hide()` code still runs, which remove the "experience" section even though the user might have a selection in there that they want to edit. The javascript needs to be aware that the Teaching/TAing checkboxes might already be checked on initial render.

**Comment by mfung** [comment]
@tjgrathwell, on edit it now shows the additional fields no matter what. I assume the purpose for this was to slim down the form for new volunteers so they engage, if a person is editing they have already decided they want to engage.

**Comment by tjgrathwell** [comment]
Hi @mfung, that implementation doesn't look ideal to me.

I think it would be hard for someone reading the template code to understand why sometimes the div needs to be `volunteer_preference_panel` and sometimes `volunteer_preference_panel_shown`.

I think it would be better if the javascript was changed to count the amount of `$(".volunteer_preference:checked")` after the form renders (just like it does after click) and use that to determine whether to show or hide.

Basically something like:

```
toggleVolunteerPreferenceVisibility = ->
  if $(".volunteer_preference:checked").length > 0
    $(".volunteer_preference_panel").show();
  else
    $(".volunteer_preference_panel").hide();

$ ->
  toggleVolunteerPreferenceVisibility()

$(document).on 'click', '.volunteer_preference', toggleVolunteerPreferenceVisibility
```

**Comment by mfung** [comment]
Hi @tjgrathwell, refactored.

**Comment by tjgrathwell** [comment]
Thanks! Deployed. :rocket:
